inc/LocalProvider.php

```diff
@@ -18,11 +18,11 @@ public function get_name() : string {
 		return apply_filters( 'amf/local_provider/name', __( 'Local Media', 'asset-manager-framework' ) );
 	}
 
-	protected function request( array $args ) : MediaList {
+	protected function request( array $args ) : MediaResponse {
 		// Call the default core attachment query AJAX handler.
 		// This will return JSON and exit before the return statement below.
 		wp_ajax_query_attachments();
-		return new MediaList();
+		return new MediaResponse( new MediaList(), 1, 1 );
```

Comment:
I know this line is not actually executed, but I'm wondering if we want to provide some defaults to the `MediaResponse` constructor? Assuming an empty list, which means a total of `0` items, and the WordPress default of `40` items per page, we can just do this:

```suggestion
		return new MediaResponse();
```